rules.d/meson.build

```
@@ -12,6 +12,7 @@ rules = files('''
         60-evdev.rules
         60-fido-id.rules
         60-input-id.rules
+        60-net-dhcp.rules
```

```suggestion
        81-net-dhcp.rules
```

If the above suggestion to use ID_NET_DRIVER comparison is taken into account, this will need to change too.